### PR TITLE
Downgrade to Go 1.16

### PR DESCRIPTION
**Pull request by matthewp**
## Changes

- Adds a failing test of a corrupt wasm file
- Downgrades to Go 1.16 which makes that test pass.

## Testing

Yep

## Docs

N/A

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ Contributions are welcome to the Go compiler!
 
 ### Go
 
-[Go][go] `1.17.x` is needed to work with this repo. On Macs, installing via [Homebrew][homebrew] is recommended: `brew install go`. For Windows & Linux, you can [follow Go’s installation guide][go] if you don’t have your own preferred method of package installation.
+[Go][go] `1.16.x` is needed to work with this repo. On Macs, installing via [Homebrew][homebrew] is recommended: `brew install go`. For Windows & Linux, you can [follow Go’s installation guide][go] if you don’t have your own preferred method of package installation.
```
Comment (drwpow) [inline CONTRIBUTING.md:9]:
```suggestion
[Go][go] `1.16.x` is needed to work with this repo. On Macs, installing via [Homebrew][homebrew] is recommended: `brew install go@1.16`. For Windows & Linux, you can [follow Go’s installation guide][go] if you don’t have your own preferred method of package installation.
```

File: cmd/astro-wasm/astro-wasm.go [hunk 2]
```diff
@@ -1,4 +1,4 @@
-//go:build js && wasm
```
Comment (drwpow) [inline cmd/astro-wasm/astro-wasm.go:1]: what does this change do?

File: lib/compiler/test/basic.test.mjs [hunk 3]
```diff
@@ -60,4 +62,4 @@ div {
   // console.log(html);
 }
 
-run();
+await run();
```
Comment (drwpow) [inline lib/compiler/test/basic.test.mjs:65]: 👍🏻

**Review by drwpow — APPROVED** [review]

File: cmd/astro-wasm/astro-wasm.go (hunk 2 above)
Comment (matthewp) [reply cmd/astro-wasm/astro-wasm.go:1]: I think this is go 1.17 syntax? Not sure totally, just that the wasm won't build with this old comment.

**Pull request merged**